File: go-sdk/pkg/models/operations/getprofileimage.go
package operations

type GetProfileImagePathParams struct {
	Username string `pathParam:"style=simple,explode=false,name=username"`
}

type GetProfileImageRequest struct {
	PathParams GetProfileImagePathParams
}

type GetProfileImageResponse struct {
	ContentType                             string
	StatusCode                              int64
	GetProfileImage200ApplicationJSONObject map[string]interface{}
}

File: ts-sdk/src/sdk/models/operations/getprofileimage.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class GetProfileImagePathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=username" })
  username: string;
}


export class GetProfileImageRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: GetProfileImagePathParams;
}


export class GetProfileImageResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  getProfileImage200ApplicationJSONObject?: Record<string, any>;
}
